actors/builtin/miner/policy.go

```diff
@@ -130,10 +124,10 @@ type VestSpec struct {
 }
 
 var PledgeVestingSpec = VestSpec{
-	InitialDelay: abi.ChainEpoch(SecondsInYear / EpochDurationSeconds),    // 1 year, PARAM_FINISH
-	VestPeriod:   abi.ChainEpoch(SecondsInYear / EpochDurationSeconds),    // 1 year, PARAM_FINISH
-	StepDuration: abi.ChainEpoch(7 * SecondsInDay / EpochDurationSeconds), // 1 week, PARAM_FINISH
-	Quantization: SecondsInDay / EpochDurationSeconds,                     // 1 day, PARAM_FINISH
+	InitialDelay: abi.ChainEpoch(7 * abi.SecondsInDay / abi.EpochDurationSeconds),   // 1 week for testnet, PARAM_FINISH
+	VestPeriod:   abi.ChainEpoch(7 * abi.SecondsInDay / abi.EpochDurationSeconds),   // 1 week for testnet, PARAM_FINISH
+	StepDuration: abi.ChainEpoch(12 * abi.SecondsInHour / abi.EpochDurationSeconds), // 12 hours for testnet, PARAM_FINISH
```

Comment:
what happens if EpochDurationSeconds doesnt evenly divide secondsInHour?

Reply:
it will be truncated which i think it's fine, we will have to add test cases here to make sure

Reply:
As long as the remainder `12*SecondsInHour % EpochDurationSeconds` is less than, say, 1 percent of `12*SecondsInHour`, it’s not so bad. But it would be really nice if the epoch duration is a divisor of 3600. There are so many: 1 2 3 4 5 6 8 9 10 12 15 16 18 20 24 25 30 36 40 45 48 50 60 72 75 80 90 100...